client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx

```diff
@@ -249,7 +249,7 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
 								) }
 							{ ! hasSevenDayRefundPeriod &&
 								translate(
-									'Simply click the link below and select the Business plan option to upgrade today {{b}}for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more{{/b}}. Once you upgrade, you’ll have 30 days to evaluate the plan and decide if it’s right for you.',
+									'Simply click the link below and select the Business plan option to upgrade today {{b}}for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more{{/b}}. Once you upgrade, you’ll have 14 days to evaluate the plan and decide if it’s right for you.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.